plugins/pulp_ostree/plugins/importers/steps.py

```diff
@@ -182,20 +195,23 @@ def process_main(self, item=None):
         """
         lib_repository = lib.Repository(self.parent.storage_dir)
         for ref in lib_repository.list_refs():
+            # the branches listed here can have an undesired prefix ending with a ":"
+            branch = ref.path.split(':')[-1]
```

Comment:
Looks like libostree has changed how it stores refs pulled from a remote in refs/ between 2015.8 and 2015.11 and this is showing up in how refs are listed.  

Pulled branches used to be stored in:
```
refs/heads/
```
but are now stored in:
```
refs/remotes/
```
This is not a big deal but did impact the `Repository.list_refs()` result.  I think your patch is exactly the correct thing to do (at least for now). 

Testing did not reveal any further ramifications.